### DISCUSSION TITLE
Verify Smart Contracts in Zksync

### DISCUSSION THREAD

**Discussion by Jjpellegrino** [Cyfrin/foundry-full-course-cu#2668] [Q&A]
I tried to verify my smart contract the same way Patrick did in the 18th video of the second course, but Remix doesn't allow me to do it. Can anyone tell me if I'm missing an extension, or has the process for verifying contracts changed?

**Comment by EngrPips** [comment]
Working with `ZkSync` isn't very smooth as of now, and we expect that to change with time, so you can move on if you run into any problem with `ZkSync` stuff.

Also, make sure you are working in the `default-workspace` in `Remix` and also ensure that your solidity files are inside the `contracts` folder or did you figure it out already?
